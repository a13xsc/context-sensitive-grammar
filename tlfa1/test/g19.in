   (        {     R    ,B         ,     H,      W         ,        Q  ,    U  ,    V        ,        A    ,     E     , P  ,       F ,     G        ,    N    , M        , Z      ,      T    , Y    ,       J       ,S,       I ,     X   ,       D    ,    K        ,C         ,          O ,          L             },    {            }                 ,   {           (          A          ,          e  ), (        E    ,          DAQO   )         ,      (          D      ,   RQRQ)          ,       (      G, e        )     ,   (         F  ,e ),    (    I   ,      AM          )   ,  (      K         ,        e    )   ,        ( J  ,   e     )     ,     (M      ,  DRAAZARDZQRQDRDA      )        ,         (      O  ,   RDZIQDZAMMIDMZMMM          ),      (     Q          ,       AAAAAAAAAAAAAAAAAAA       )   ,   (P   ,         e   ) , (   R ,     e        ) ,         (        T         ,       TWA         )    ,   (          T         ,     ZIRIRQ )   ,   (W ,    e       )         ,      (      V   ,          e      )        ,    (   Y          , e     )      ,          (  X   , e     )        ,(    Z      ,        DA        )         }     ,         H   )        
