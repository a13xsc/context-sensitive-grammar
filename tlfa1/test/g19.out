L
N
S
H
U
C
B
